src/core/config/createCoreConfigs.js

```diff
@@ -31,5 +31,6 @@ export default () => ({
   orgId: string()
     .unique()
     .required(),
+  onBeforeLinkCollect: callback().default(noop),
```

Comment:
It kind of bugs me we're using `Collect` here and `Send` in `onBeforeEventSend`. I suppose that's because we have a config option called `clickCollectionEnabled` and are aligning `onBeforeLinkCollect` with that.  Maybe `onBeforeLinkClickSend` would be better? Or maybe even `onLinkClick`? I'm curious to hear what you or others think.